### PR TITLE
Fix openssl ca, to correctly make output file binary when using -spkac

### PR DESCRIPTION
**Pull request by levitte**
On Unix, this doesn't matter, but on other platforms, it may.

### CODE REVIEW

File: apps/ca.c [hunk 1]
```diff
@@ -722,8 +722,13 @@ int ca_main(int argc, char **argv)
 
     /*****************************************************************/
     if (req || gencrl) {
+        if (spkac_file != NULL) {
+            output_der = 1;
+            batch = 1;
+        }
         /* FIXME: Is it really always text? */
-        Sout = bio_open_default(outfile, 'w', FORMAT_TEXT);
+        Sout = bio_open_default(outfile, 'w',
```
Comment (mattcaswell) [inline apps/ca.c]: It seems like the FIXME can now be removed?

Comment (levitte) [reply apps/ca.c]: Haha, yes!  How did I miss that?

**Review by mattcaswell — APPROVED** [review]

**Comment by mattcaswell** [comment]
Oh, wait hold for a second.

**Comment by levitte** [comment]
Hmm?

**Comment by mattcaswell** [comment]
Just needed to check something. But I checked, and its fine. Approval stands!

**Comment by levitte** [comment]
Cool

**Comment by levitte** [comment]
Merged.

master / 1.1.1:
25642ad29e6a2c15c10ceb5e4f029638f73a879e Fix openssl ca, to correctly make output file binary when using -spkac

1.1.0:
29a7148d4e47cf5c7861ed66679d69d5380c95f2 Fix openssl ca, to correctly make output file binary when using -spkac

**Pull request closed without merging**